.husky/pre-commit

```diff
@@ -1,4 +1,6 @@
 npm run format
 npm run lint
 npm run leaks
-npm test
\ No newline at end of file
+npm test
+(cd examples/es6 && npm i && npm run build)
+(cd examples/commonjs && npm i)
```

Comment:
Why do we want to run it before commit?

Reply:
automatically modify the package lock if needed before commit